src/app/admin/modules/data-management/components/data-graph/data-graph.component.html

```diff
@@ -0,0 +1,66 @@
+<div class="container-fluid">
+  <div class="row">
+      <div class="col-md-2">
+        <mat-form-field>
+          <mat-label>选择统计类型</mat-label>
+          <mat-select [formControl]="selectedGraphType">
```

Comment:
I wonder why you didn't follow the official demo, since you are using `FormGroup`, instead of multiple separate `FormControl`.